### ISSUE TITLE
Cannot read property 'toLowerCase' of undefined during error propagation

### ISSUE THREAD

**Issue by robrigo** [eosdac/ual-wax#3]
I'm not sure if this issue is originating on the `ual-wax` side or `ual-reactjs-renderer` so I'm reporting it here just in case. The stack trace would indicate this is a problem with ual-wax propagating an error without a type set.

```
Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
    at Function.boxTitle (UALBoxParts.js:251)
    at UALBoxBase.render (UALBox.js:205)
    at finishClassComponent (react-dom.development.js:17160)
    at updateClassComponent (react-dom.development.js:17110)
    at beginWork (react-dom.development.js:18620)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
    at invokeGuardedCallback (react-dom.development.js:292)
    at beginWork$1 (react-dom.development.js:23203)
    at performUnitOfWork (react-dom.development.js:22157)
    at workLoopSync (react-dom.development.js:22130)
    at performSyncWorkOnRoot (react-dom.development.js:21756)
    at react-dom.development.js:11089
    at unstable_runWithPriority (scheduler.development.js:653)
    at runWithPriority$1 (react-dom.development.js:11039)
    at flushSyncCallbackQueueImpl (react-dom.development.js:11084)
    at flushSyncCallbackQueue (react-dom.development.js:11072)
    at scheduleUpdateOnFiber (react-dom.development.js:21199)
    at Object.enqueueSetState (react-dom.development.js:12639)
    at UALProvider.push../node_modules/ual-reactjs-renderer/node_modules/react/cjs/react.development.js.Component.setState (react.development.js:471)
    at broadcastStatus (UALProvider.js:148)
    at _callee$ (UALProvider.js:179)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:274)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at asyncGeneratorStep (UALProvider.js:13)
    at _next (UALProvider.js:13)
```

**Comment by robrigo** [comment]
To produce this bug, I added the ual-wax plugin to ual-reactjs-renderer and attempted to login. The login step works (a WaxUser is returned) but attempting to sign with that user after to prove the identity request fails.

**Comment by robrigo** [comment]
The page redirects to this error screen with some additional details:

<img width="1725" alt="Screen Shot 2020-06-29 at 9 13 19 PM" src="https://user-images.githubusercontent.com/590263/86141344-f29b3800-babf-11ea-8a13-aaeed29c112e.png">

**Comment by michaeljyeates** [comment]
Fixed in 1.2.2
